### PR TITLE
#320 Fix ApiProduct Entity setProxies() runtime error

### LINKED ISSUE #320: Missing splat operator
**Describe the bug**
A missing splat operator is causing an exception to be thrown when calling the `setProxies()` method on a `\Drupal\apigee_edge\Entity\ApiProxy` object

**To Reproduce**
1. Create an instance of `\Drupal\apigee_edge\Entity\ApiProxy`
2. Call the `setProxies()` method with any string value as a parameter

**The Fix**
In the `setProxies()` method of `\Drupal\apigee_edge\Entity\ApiProxy`, add a splat operator to the call to `$this->decorated->setProxies()`
```php
public function setProxies(string ...$proxy): void {
  $this->decorated->setProxies(...$proxy);
}
```

### PR DESCRIPTION
**Pull request by cnovak**
Add splat operator and tests

Fixes #320

### CODE REVIEW

File: tests/src/Unit/Entity/ApiProductTest.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Test ApiProductTest class.
+ *
+ * @group apigee_edge
+ */
+class ApiProductTest extends UnitTestCase {
+
+  /**
```
Comment (shadcn) [inline tests/src/Unit/Entity/ApiProductTest.php:30]: Missing short description here.

**Review by shadcn — CHANGES_REQUESTED** [review]
Just a minor phpcs nit. Otherwise this is green to go.

**Comment by cnovak** [comment]
@arshad PR ready to review again, fixed phpcs issue

**Review by shadcn — APPROVED** [review]
Looks good. Approved. Thanks @cnovak

**Pull request merged**